scripts/update-version.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+NEW_CANARY_VERSION="$(grep -o '\"version\": \"[[:digit:]].[[:digit:]].[[:digit:]]-canary.[[:digit:]]\"' package.json | awk -F. '{$NF = $NF + 1;} 1' OFS=.)\""
```

Comment:
what's the expected input/outputs here?

Reply:
"version": "0.6.3-canary.1" => "version": "0.6.3-canary.2"

Do we need to update the patch version rather than the canary version?  or as well?

Reply:
nope, think this is good. at a minimum it prevents us overwriting a previous release which is the goal